apps/art/serializers.py

```diff
@@ -1,8 +1,19 @@
 from art.models import Art
+from art.models import Artist
 from rest_framework import serializers
 
 
 class ArtSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Art
+<<<<<<< HEAD
         fields = ('name', 'description', 'created', 'rating', 'image','timeframe_for_order', 'price', 'availability')
+=======
+        fields = ('name', 'description', 'created', 'rating', 'image','timeframe_for_order', 'price')
```

Comment:
I think we somehow overwrote a field in the process of merging this.  We need to add availability back in:
```suggestion
        fields = ('name', 'description', 'created', 'rating', 'image','timeframe_for_order', 'price', 'availability')
```